files/en-us/web/api/domrectreadonly/fromrect/index.html

```diff
@@ -18,20 +18,20 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="syntaxbox notranslate">var domRect = DOMRectReadOnly.fromRect(rectangle)</pre>
+<pre class="syntaxbox notranslate">let domRect = DOMRectReadOnly.fromRect(<var>rectangle</var>)</pre>
 
 <h3 id="Parameters">Parameters</h3>
 
 <dl>
- <dt>rectangle {{optional_inline}}</dt>
- <dd>An object specifying the location and dimensions of a rectangle. All properties default to <code>0</code>. The properties are:</dd>
- <dd>
- <ul>
-  <li><code>x</code>: The coordinate of the left side of the rectangle.</li>
-  <li><code>y</code>: The coordinate of the top side of the rectangle.</li>
-  <li><code>width</code>: The width of the rectangle.</li>
-  <li><code>height</code>: The height of the rectangle.</li>
- </ul>
+  <dt><code><var>rectangle</var></code> {{optional_inline}}</dt>
+ <dd>An object specifying the location and dimensions of a rectangle. All properties default to <code>0</code>. The properties are:
```

Comment:
Convention has it nested extra options, or object properties such as this are held in a `ul` within the `dd` - however this does highlight the original `ul` was not inside the `dd`

```suggestion
<dd>An object specifying the location and dimensions of a rectangle. All properties default to <code>0</code>. The properties are:
  <ul>
     <li><code>x</code>: The coordinate of the left side of the rectangle.</li>
     <li><code>y</code>: The coordinate of the top side of the rectangle.</li>
     <li><code>width</code>: The width of the rectangle.</li>
     <li><code>height</code>: The height of the rectangle.</li>
   </ul>
  </dd>